egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -347,4 +347,164 @@ def AddClstmLayer(config_lines,
             }
 
 
+def AddBlstmLayer(config_lines,
```

Comment:
Could you please write the equations you are implementing on the top of the component. It would help me.